indigo_app/static/javascript/indigo/views/work_commencements.js

```diff
@@ -7,18 +7,24 @@
   /**
    * Handle the work commencements display.
    */
+
```

Comment:
The class comment should be adjacent to the class it documents.

Reply:
done